draft-amend-tsvwg-multipath-dccp.mkd

```diff
@@ -154,7 +197,7 @@ does not provide built-in support for those multiple application subflows.
 In the following, use of the term subflow will refer to physical
 separate DCCP subflows transmitted via different paths, but not to
 application subflows. Application subflows are differing content-wise by
-source and destination application as e.g. enabled by Service Codes
+source and destination application as, e.g., enabled by Service Codes
```

Comment:
I don't fully understand this point, but would it read better like this:
"Application subflows are differing content-wise by source and destination application as, for example, enabled by Service Codes introduced to DCCP in {{RFC5595}}, and can be multiplexed over a single DCCP connection."

i.e. I'm not sure if the "can be multiplexed" part relates to the example, or the start of the sentence. (Note the comma after the rfc5595)

Reply:
@dirkhugo can you check?

Reply:
'can be multiplexed' relates to 'application subflows' - thus should relate to the start of the sentence ... I think adding 'port' makes it clearer:
"Application subflows are differing content-wise by source and destination port per application as, for example, enabled by Service Codes introduced to DCCP in {{RFC5595}}, and those subflows can be multiplexed over a single DCCP connection." - thanks!

Reply:
Taken in b2a1befb205368fe8accf283406dd98bc544170e